_includes/pbc/all/upgrade-modules/upgrade-the-productconfigurationsrestapi-module.md

```diff
@@ -1,17 +1,21 @@
+## Upgrading from version 0.2.* to version 1.0.*
 
+*Estimated migration time: 5 minutes*
 
-## Upgrading from version 0.1.* to version 0.2.*
+`ProductConfigurationsRestApi` v1.0.0 introduces the following backward incompatible changes:
 
-*Estimated migration time: 5 minutes*
+* Replaced ProductConfigurationStorageClientInterface::findProductConfigurationInstanceBySku() with ProductConfigurationStorageClientInterface::getProductConfigurationInstanceCollection().
 
-To upgrade the `ProductConfigurationsRestApi` module from version 0.1.* to version 0.2.*, do the following:
+To upgrade the `ProductConfigurationsRestApi` module from version 0.2.* to version 1.0.*, do the following:
```

Comment:
I can see that in this document we replace the 0.1 to 0.2 migration with 0.2 to 1.0, while in the `_includes/pbc/all/upgrade-modules/upgrade-the-productconfiguration-module.md` we added a section keeping the old migration section. Please mmake these two changes consistent and either leave both or replace them in all the cases.